Tests/SonarScanner.MSBuild.PreProcessor.Test/EmbeddedAnalyzerInstallerTests.cs

```diff
@@ -139,29 +124,31 @@ public void EmbeddedInstall_TempPath_Succeeds()
         [TestMethod]
         public void EmbeddedInstall_MultiplePlugins_Succeeds()
         {
+            const string p1Resource1 = "p1.resource1";
+            const string p1Resource2 = "p1.resource2";
+            const string p2Resource1 = "p2.resource1";
             // Arrange
             var localCacheDir = TestUtils.CreateTestSpecificFolderWithSubPaths(TestContext);
             var logger = new TestLogger();
 
-            var request1 = new Plugin("no.matching.resource.plugin", "2.0", "non.existent.resource.zip");
```

Comment:
Removed: no matching plugin is causing hard failure (already tested in `EmbeddedInstall_MissingResource_ThrowFileNotFoundException`).